sp/src/game/server/ez2/npc_basepredator.cpp

```diff
@@ -25,6 +25,9 @@
 
 ConVar ai_debug_predator( "ai_debug_predator", "0" );
 ConVar sv_predator_heal_render_effects( "sv_predator_heal_render_effects", "1", FCVAR_NONE, "Should predators like bullsquids turn green after healing?" );
+#ifdef EZ2
+ConVar npc_predator_obstruction_behavior( "npc_predator_obstruction_behavior", "1" );
```

Comment:
Per the other review, if we default that one to 0 then this one should also be 0